feeds/wlan-ap/opensync/src/platform/openwrt/src/command/src/node_config.c

```diff
@@ -352,12 +364,12 @@ static void led_handler(int type,
 				sscanf(sysled,"/%[^/]/%[^/]/%[^/]/%s", sys, class, leds, led_string);
 				sscanf(led_string,"%[^:]:%[^:]:%s",ap_name, color, led_section);
 				if(available_led_check(led_section)) {
-					blob_buf_init(&b, 0);
-					blobmsg_add_string(&b, "sysfs", led_string);
-					blobmsg_add_string(&b, "value", "on");
-					blobmsg_add_string(&b, "key", conf->key );
-					blobmsg_add_string(&b, "trigger", "heartbeat");
-					blob_to_uci_section(uci, "system", led_section, "led", b.head, &led_param, NULL);
+					if(!strcmp(conf->key, "led_blink")) {
+						set_led_config("heartbeat", conf->key, conf->value, led_string, led_section);
+					}
+					if(!strcmp(conf->key, "led_off")) {
```

Comment:
Maybe change this to an `else if`
```suggestion
					else if(!strcmp(conf->key, "led_off")) {
```

Reply:
sure, I'll do that